requirements.txt

```diff
@@ -3,7 +3,8 @@ ipython>=5.5.0
 ipywidgets>=7.6.5  # required by pycaret.internal.display
 tqdm>=4.62.0  # required by pycaret.internal.display
 numpy>=1.21, <1.27
-pandas>=1.3.0, <2.0.0  # Can't >=2.0.0 because of sktime
+pandas>=1.3.0, <2.1.0
```

Comment:
Why <2.1.0, can't we  make it support the latest version as well? If the problem is the matplotlib version, try checking what the issue is there and bump matplotlib as well.

Reply:
initially I thought about supporting pandas 2.1, but it seems that many problems can arise with third-party packages and in testing, so I thought I would do it in a phased manner, a pull-request for pandas 2.0 and once approved, make another pull-request for pandas 2.1.